rdbms/src/main/java/io/spine/server/storage/jdbc/IdColumn.java

```diff
@@ -167,35 +180,86 @@ public void setId(int index, Integer id, PreparedStatement statement)
      * Helps to work with columns which contain either {@link Message} or {@code string}
      * {@link Entity} IDs.
      */
-    private static class StringOrMessageIdColumn<I> extends IdColumn<I> {
+    private abstract static class StringOrMessageIdColumn<I> extends IdColumn<I> {
 
         private StringOrMessageIdColumn(String columnName) {
             super(columnName);
         }
 
         @Override
-        public Sql.Type getColumnDataType() {
-            return VARCHAR_255;
+        public Sql.Type getSqlType() {
+            return Sql.Type.VARCHAR_255;
         }
 
         @Override
         public void setId(int index, I id, PreparedStatement statement) throws DatabaseException {
-            final String idString = idToString(id);
+            final String idString = normalize(id);
             try {
                 statement.setString(index, idString);
             } catch (SQLException e) {
                 throw new DatabaseException(e);
             }
         }
+
+        /**
+         * Normalizes the identifier before setting it to a {@link PreparedStatement}.
+         *
+         * <p>The method may perform a {@code String} conversion, validation or no action for
+         * a {@code String} input.
+         *
+         * @param id the identifier to convert normalize
```

Comment:
Something with the grammar in this comment. Probably it should be `the identifier to normalize`.